### PR TITLE
Improved tweet cards

### PR DESCRIPTION
**Pull request by mrhcjones**
* save button -> like button #108 
* shrinking of tweet cards (similar to twitdere) #12 
* Link copy #98

**Testing APKS**
https://github.com/mrhcjones/Quacker/actions/workflows/ci.yml

### CODE REVIEW

File: lib/tweet/tweet.dart [hunk 1]
```diff
@@ -298,17 +297,23 @@ class TweetTileState extends State<TweetTile> with SingleTickerProviderStateMixi
     });
   }
 
-  static const List<String> _twitterUrls = ['twitter.com', 'pic.twitter.com', 'twimg.com', 'abs.twimg.com', 'pbs.twimg.com', 'video.twimg.com'];
+  static const List<String> _twitterUrls = [
+    'twitter.com',
```
Comment (williamdes) [inline lib/tweet/tweet.dart:301]: I saw today a x.com short link on a twitter post

Comment (mrhcjones) [reply lib/tweet/tweet.dart:301]: I've added x.com to the list of urls, anything else needing to be changed?

**Review by williamdes — APPROVED** [review]

**Comment by williamdes** [comment]
I found a small bug, click on a twitt and like it, go back to the main feed
The tweet is not liked

Reddit also has this bug..

**Comment by j-fbriere** [comment]
@mrhcjones If you want me to include your changes, at least make sure to your code is formatted properly so that I don't have to take hours to reformat it to check the actual differences.
Also explain what is the point of changing Save Tweet by Like Tweet? What is the added value?
Also there are 3 different features in the same pull request. That is not ok.
Please create 3 different pull requests for those 3 different features.

**Comment by mrhcjones** [comment]
> @mrhcjones If you want me to include your changes, at least make sure that your code is formatted properly so that I don't have to take hours to reformat it to check the actual differences. Also explain what is the point of changing Save Tweet by Like Tweet? What is the added value? Also there are 3 different features in the same pull request. That is not ok. Please create 3 different pull requests for those 3 different features. It's then easier for me to verify, integrate and test each feature. Thank you.

Thank you for the feedback, I'll separate parts of the pull request into their respective categories. Having a like tweet button instead of a save tweet one makes it easier for new users to understand.

Thank you for taking the time to look at this request anyways.
Have a great rest of your week! 😀
- Harley

**Pull request closed without merging**